documentation/docs/verifiable_credentials/verifiable_presentations.mdx

```diff
@@ -10,17 +10,157 @@ keywords:
+
+The framework checks:
+
+- **Semantic structure**: ensures the presentation and its credentials adhere to the specification.
+- **Presentation proof**: verifies the presentation signature against the DID Document of the holder.
+- **Credential proofs**: verifies the credential signatures against the DID Documents of their respective issuers.
+- **Optional validations**: additional checks on signatures and credential fields can be configured by the verifier.
+
+Note that a verifier may specify which DID Documents to use for the holder and issuers, otherwise they are resolved from the IOTA Tangle automatically.
+
+Currently, the following are *not* checked automatically:
+
+- **Data schemas**: credentials that specify a [schema](https://www.w3.org/TR/vc-data-model/#data-schemas) property should be examined to ensure conformance.
+- **Fitness for purpose**: whether the credentials in a presentation and the data within them are acceptable and valid depends on the context in which they are used. Verifiers should ensure that the credential types, subjects, and schemas sent by a holder match what was requested.
+- **Issuer trustworthiness**: verifiers must check that they trust the issuer on each individual credential in a presentation. The framework only verifies that the issuer's signature on each credential is current and valid against the given options.
+
+The default validation behaviour may be modified by the following options.
+
+### Proof Verifier Options {#proof-verifier-options}
+
+While the framework always verifies that the digital signature on a verifiable presentation is valid, a verifier may validate additional fields in the proof on a presentation.
+Notably, to mitigate potential replay attacks a verifier should always check that the challenge and domain fields match what was sent to the holder when requesting the presentation.
+See [Security Considerations](#security-considerations) for more information.
+
+The following options are available:
+
+- **Method Scope**: check the signing verification method has a particular [verification relationship](https://www.w3.org/TR/did-core/#verification-relationships). Overridden by the proof purpose check.
+- **Method Type**: check the signing verification method has a particular type.
+- **Challenge**: check the challenge field matches this string.
+- **Domain**: check the domain field matches this string.
+- **Proof Purpose**: require a specific purpose on the proof. See [Proof Options](#proof-options).
+- **Allow Expired**: accept proofs where the current datetime is after their expiration. Default is to reject expired proofs.
+
+See [Proof Options](#proof-options) for more information on setting these properties as a holder when signing a verifiable presentation.
+
+### Subject-Holder Relationship {#subject-holder-relationship}
+
+Specifies the expected relationship between the holder that signed the verifiable presentation and the subject specified in each verifiable credential.
```

Comment:
Perhaps we can already add an explanation of the `nonTransferable` property here at the top as that is the most frequent key thing required to understand this section.

Reply:
Changed that section to read as follows:

> Specifies the expected relationship between the holder that signed the verifiable presentation and the subject specified in each verifiable credential. This can be restricted by the [`nonTransferable`](https://www.w3.org/TR/vc-data-model/#nontransferable-property) property, which indicates that a verifiable credential must only be encapsulated into a verifiable presentation whose holder matches the credential subject.
> 
> By default, the framework always enforces that the holder matches the subject on all credentials. The following options are available to modify that behaviour: